### ISSUE TITLE
Add Django Backend

### ISSUE THREAD

**Issue by muntaxir4** [muntaxir4/heliotask#2]
- Create `django` folder and provide the codes for all routes
[ Refer to other backend codes for info ]

**Comment by Revant-S** [comment]
@muntaxir4 can you please assign this issue  to me . I am highly interested in backend related work and have experience in working with different python frameworks

**Comment by muntaxir4** [comment]
@Revant-S Yeah sure, you can work on implementing the Django backend. And feel free to ask any related questions.
